ocs_ci/ocs/node.py

```diff
@@ -384,7 +384,15 @@ def add_new_node_and_label_upi(
     plt = PlatformNodesFactory()
     node_util = plt.get_nodes_platform()
     node_util.create_and_attach_nodes_to_cluster(node_conf, node_type, num_nodes)
-    for sample in TimeoutSampler(timeout=600, sleep=6, func=get_worker_nodes):
+
+    addnodetimeout = 600
+
+    if config.ENV_DATA["platform"].lower() == constants.IBM_POWER_PLATFORM:
+        addnodetimeout = 1200
+
+    for sample in TimeoutSampler(
+        timeout=addnodetimeout, sleep=6, func=get_worker_nodes
```

Comment:
I didn't think of it before... but why the time to sleep here is only 6 seconds? If we wait 600 or 1200 seconds, I thought it would be 20 or even 30 seconds.

Reply:
@yitzhak12 The code I am proposing is just doubling the timeout (600 to 1200) if the environment is IBM _POWER_PLATFORM. Is there an issue with the original code itself? Then I can fix it, let me know.

Reply:
Sorry for the late response. Yes, I know that you took the timeout from the previous code - That's why I mentioned that I didn't think of it before. Maybe we need to ask another one in the relevant field about the matter. I think that sleep for 20 or 30 seconds when we wait for 600 seconds or more would be enough. But maybe there was a reason for waiting only 6 seconds between the iterations.